app/client/views/dashboard/dashboardCtrl.js

```diff
@@ -103,5 +115,64 @@ angular.module('reg')
               });
         });
       };
+      
+      // Ask for location --------------------------------------------------
+      $scope.onCheckIn = function () {
+        if (navigator.geolocation) {
+          navigator.geolocation.getCurrentPosition(
+              ({ coords }) =>  {
+                const { latitude, longitude } = coords;
+                console.log(latitude, longitude);
+                var isCheckedIn = UserService
+                    .makeCheckIn(latitude, longitude)
+                    .success(function (response) {
+                      console.log(response);
+                    });
+                if ((isCheckedIn) ) {
+                  () => f;
+                  /*
+                  botonRegistro.toggleClass('hiddendiv');
+                  botonSalida.toggleClass('hiddendiv');
+                   */
+                  console.log(status);
+                } else {
+                  alert('Lo sentimos, no puedes inicar sesión si no estás en la ubicacion del evento');
+                }
+              });
+          // this.hasCurrentSession = false;
+        } else {
+          alert('Geolocation is not supported by this browser.');
+        }
+      };
 
+      // Ask for team mates ------------------------------------------------
+      function _populateTeammates() {
+        UserService
+            .getMyTeammates()
+            .success(function(users){
+              $scope.error = null;
+              $scope.teammates = users;
+              _askForTable(users);
+            });
+      }
+      
+      // Ask for table number ----------------------------------------------
+      function _askForTable(users) {
+        //if the users haven't been assing a table
+        if(!users.assign) {
+          for(let teammate of users.teammates) {
```

Comment:
Since assign is telling you if the amount of team members checked in is valid in order to assign a table, you don't have to iterate through the users which are checked in. Also it is not taking into account the `teamSizeAccepted` received from the server.